build/mac/build.sh

```diff
@@ -1,16 +1,65 @@
 #!/bin/bash
 
-# On M1 mac below would be DESTINATION="arm64_mac_sqlite"
-DESTINATION="$(uname -m)_mac_sqlite"
+# Current directory
```

Comment:
Quite a few changes in this file, basically to allow building for different architecture